files/Trimble_Tekla/Sprint_1_3.ifc

```diff
@@ -0,0 +1,67 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition[Undefined]'),'2;1');
+FILE_NAME('Sprint_1_3.ifc','2023-03-30T11:17:05',('Author/creator of the IFC file'),('Trimble Inc.'),'IfcExportSDK2 ver. 2.3.4','Trimble Software - Version xx.x','Authorizer of the IFC file');
+FILE_SCHEMA(('IFC4X4'));
+ENDSEC;
+
+DATA;
+#1= IFCPOSTALADDRESS(.HOME.,'My postal address',$,$,('StreetAddress 1','StreetAddress 2'),'Box 1','Town','Region','Box 1','Country');
+#2= IFCACTORROLE(.CONSULTANT.,'Role','Role Description');
+#3= IFCPERSON('HLahtel','Family name','Given Name',$,$,$,(#2),(#1));
+#4= IFCACTORROLE(.CONSULTANT.,$,$);
+#5= IFCPOSTALADDRESS(.OFFICE.,'Office postal address',$,$,('StreetAddress 1','StreetAddress 2'),'Box 1','Town','Region','Box 1','Country');
+#6= IFCORGANIZATION('TRMB','Trimble Inc.','Trimble consulting',(#4),(#5));
+#7= IFCPERSONANDORGANIZATION(#3,#6,$);
+#8= IFCAPPLICATION(#6,'version','Application full name','Application Identifier');
+#9= IFCOWNERHISTORY(#7,#8,$,.NOCHANGE.,$,$,$,1680164225);
+#10= IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#11= IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#12= IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#13= IFCSIUNIT(*,.MASSUNIT.,.KILO.,.GRAM.);
+#14= IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
+#15= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#16= IFCSIUNIT(*,.SOLIDANGLEUNIT.,$,.STERADIAN.);
+#17= IFCSIUNIT(*,.POWERUNIT.,$,.WATT.);
+#18= IFCUNITASSIGNMENT((#10,#11,#12,#13,#14,#15,#16,#17));
+#19= IFCCARTESIANPOINT((-200.,200.,0.));
+#20= IFCDIRECTION((0.,0.,1.));
+#21= IFCDIRECTION((1.,0.,0.));
+#22= IFCAXIS2PLACEMENT3D(#19,#20,#21);
+#23= IFCDIRECTION((0.725975,0.687721));
+#24= IFCGEOMETRICREPRESENTATIONCONTEXT('3D','Model',3,1.E-06,#22,#23);
+#25= IFCPROPERTYSINGLEVALUE('Project data',$,IFCLABEL('Project data 1.'),$);
+#26= IFCPROPERTYSINGLEVALUE('Length property',$,IFCLENGTHMEASURE(1.),#10);
+#27= IFCPROPERTYSET('3XFRdIOhv9d8uYiJsG8C5a',#9,'Project pset',$,(#25,#26));
+#28= IFCARCHELEMENTTYPE('10f9A2VKb4n8rlfqboBMo$',#9,'Name',$,$,$,$,$,$,$);
+#29= IFCPROJECT('3KF6hmUH9ExfmDglggWKPu',#9,'Project name','\X2\987976EE7B804ECB\X0\',$,$,$,(#24),#18);
+#30= IFCRELDECLARES('17g0VCjnX7l8YBHhyLQpK1',#9,$,$,#29,(#28));

```

Comment:
Assuming that IfcPropertySetTemplate (`#54`) should be in the declarations as well.